platforms/android/src/org/apache/cordova/facebook/ConnectPlugin.java

```diff
@@ -299,6 +301,21 @@ public void onCompleted(GraphUser user, Response response) {
                 }
             }
             return true;
+        } else if (action.equals("getApplicationSignature")) {
+            String sig = Settings.getApplicationSignature(webView.getContext());
+            if (sig == null) {
+              callbackContext.error("Could not determine signature.");
+            } else {
+              // strip the linefeed..
+              sig = sig.replace(System.lineSeparator(), "");
```

Comment:
I think your Java indents are 2 spaces. Can you match the existing style? Thanks.